pkg/package-server/provider/registry.go

```diff
@@ -289,9 +289,11 @@ func toPackageManifest(pkg *api.Package, client registryClient) (*operators.Pack
 	catsrc := client.source
 	manifest := &operators.PackageManifest{
 		ObjectMeta: metav1.ObjectMeta{
-			Name:              pkg.GetName(),
-			Namespace:         catsrc.GetNamespace(),
-			Labels:            catsrc.GetLabels(),
+			Name:      pkg.GetName(),
+			Namespace: catsrc.GetNamespace(),
+			Labels: labels.CloneAndAddLabel(
```

Comment:
Why clone twice?

Reply:
So it can all be done in-place without extra variables. There isn't a `CloneAndAddLabels()` function.